techmatters/flex-plugins
# External Audio Recordings with Styling and Error handling

Primary reviewer:

## Description

- This PR includes changes from other work already completed. Specifically, the following: 
  - https://github.com/techmatters/flex-plugins/pull/1598
  - https://github.com/techmatters/flex-plugins/pull/1620
  - https://github.com/techmatters/flex-plugins/pull/1619
- Styling is one according to mocks: https://marvelapp.com/prototype/e285de8
- Error Handling is similar to transcripts but does not handle both 'permanent' and 'temporary' errors

### Checklist
- [n/a] Feature flags added

### Related Issues
Fixes # https://tech-matters.atlassian.net/browse/CHI-1348 & https://tech-matters.atlassian.net/browse/CHI-1349
- https://marvelapp.com/prototype/e285de8

### Verification steps
- Start a call and accept it. Save the contact or complete the task
- Search for the newly created contact
- Check the Recording section within this contact and test for the correct behavior of the Chrome audio player. The mocks are here: https://marvelapp.com/prototype/e285de8 
- _Optional_ Ensure external transcripts are still working fine for chat contacts
